server/testkit/tests/proto_v1_test.rs

```diff
@@ -1793,3 +1922,21 @@ async fn test_server_user_auth_privileged_shortcut(rsclient: KanidmClient) {
         }
     }
 }
+
+// wanna test how long it takes for testkit to start up? here's your biz.
+// turns out  as of 2023-10-11 on my M2 Max, it's about 1.0 seconds per iteration
```

Comment:
That's surprisingly fast contrast to other IDM's I've worked on

Reply:
Yeah, I was trying to work out how much effort I should put into trying to optimise it while adding more invocations of it ^_^ It's probably worth investigating at some point, but for now it's a known-known.

Reply:
Generally any performance improvement here will probably have a benefit all over the server, so I likely need to do some optimisation at some point.